man/man8/zpoolprops.8

```diff
@@ -320,6 +320,53 @@ to the enabled state.
 See
 .Xr zpool-features 5
 for details on feature states.
+.It Sy features Ns = Ns Ar filename Ns | Ns Sy all Ns | Ns Sy none
+Specifies a set of requested features for this pool. When set to
+.Sy all
+(or unset); all features are requested; when set to
+.Sy none Ns ;
+no features are requested. When set to a filename - either
+an absolute path, or relative to
+.Pa /etc/zfs/features.d Ns ,
+the list of requested features is read from that file, separated by
+whitespace and/or commas. The file may include comments; any text
+from # to the end of the line is ignored.
+.Pp
+The requested features are applied when a pool is created using
+.Sy zpool create
+and controls which features are enabled when using
+.Sy zpool upgrade Ns .
+.Sy zpool status
+will not show a warning about disabled features which are not part
+of the requested feature set.
+.Pp
+.Sy EXAMPLE
+.Bd -literal -offset indent
+.Sy # /etc/zfs/features.d/grub
+# Features compatible with grub2
+async_destroy
+bookmarks
+embedded_data
+empty_bpobj
+enabled_txg
+extensible_dataset
+filesystem_limits
+hole_birth
+large_blocks
+lz4_compress
+spacemap_histogram
+userobj_accounting
```

Comment:
`userobj_accounting` is not actually compatible with GRUB in practice.

Reply:
@rlaager Fair; I guess I was working from an older version of your document. Will remove that from examples before merge.